File: app/src/main/java/com/kfc/restorater/RestoApplication.java
package com.kfc.restorater;

import android.app.Application;
import android.app.NotificationChannel;
import android.app.NotificationManager;

public class RestoApplication extends Application {
    private static final String CHANNEL_ID = "login_notification_channel";
    private static NotificationManager notificationManager;

    @Override
    public void onCreate() {
        super.onCreate();
        NotificationChannel channel = new NotificationChannel(CHANNEL_ID, "Login Notification", NotificationManager.IMPORTANCE_HIGH);
        notificationManager = getSystemService(NotificationManager.class);
        notificationManager.createNotificationChannel(channel);
    }

}

File: app/src/main/java/com/kfc/restorater/RestoApplication.kt
package com.kfc.restorater

import android.app.Application
import android.app.NotificationChannel
import android.app.NotificationManager

class RestoApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        val channel = NotificationChannel(
            CHANNEL_ID,
            "Login Notification",
            NotificationManager.IMPORTANCE_HIGH
        )
        getSystemService(NotificationManager::class.java)
            .createNotificationChannel(channel)
    }

    companion object {
        const val CHANNEL_ID = "login_notification_channel"
    }
}